hub/powertoys/cmd-not-found.md

```diff
@@ -0,0 +1,36 @@
+---
+title: PowerToys Command Not Found for Windows
+description: A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+ms.date: 02/01/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+
+![AlwaysOnTop screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
+ - [PowerShell 7](https://learn.microsoft.com/en-us/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
```

Comment:
```suggestion
 - [PowerShell 7](/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
```

Is the `?view=PowerShell-7.4` query param needed, or can we let it default to the latest PowerShell docs?